home/models.py

```diff
@@ -323,7 +335,7 @@ def clean(self, value):
                 f"{len(result['message'])} characters long"
             )
 
-        list_items = result["list_items"]
+        list_items = [item.value["title"] for item in result["list_items"]]
```

Comment:
We should test if we still need this validation, and maybe remove it if not? With the new error translation put in place for import, I think this is no longer necessary, since we specify the max char length on the streamfield definition.